CASSANDRA.md

```diff
@@ -16,7 +16,7 @@ All examples have been properly tested, and they should work out of the box.
 
 ## Prerequisites ##
 
-The keyspace and column family needed for all examples are defined in this CQL script:
+The keyspace and column family needed for all examples are defined in this CQL script. This step can be entirely skipped in Phantom were schema is auto-generated.
```

Comment:
Feel free to move the comparison to the specific sections about Quill and Phantom. I'm not convinced that this is the correct place for this information.